extensions/adobe/experience/marketo/opportunity-marketo.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/opportunity-marketo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/context/opportunity-crm"],
+  "description": "Marketo specific Opportunity attributes.",
+  "definitions": {
+    "opportunity-marketo": {
+      "properties": {
+        "marketo:mktoGuid": {
+          "title": "Marketo GUID ID",
+          "description": "Marketo GUID ID.",
+          "type": "string"
+        },
+        "marketo:instanceID": {
+          "title": "Instance Identifier",
+          "description": "Instance identifier.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "allOf": [
```

Comment:
Don't we need to add "https://ns.adobe.com/xdm/context/opportunity-crm" here as its referenced in the "meta:extends"

Reply:
Yes, that's correct. Fix this issue. also, we need to figure how if the user adds this mixin the CRM-mixin should be greyed out in the UI.